internal/db/db.go

```diff
@@ -51,11 +52,14 @@ const (
 		VALUES ($1, $2, CURRENT_TIMESTAMP, $3, $4, $5)`
 
 	sqlGetComposes = `
-		SELECT job_id, request, created_at, image_name
-		FROM composes
-		WHERE org_id=$1 AND CURRENT_TIMESTAMP - created_at <= $2 AND deleted=FALSE
-		ORDER BY created_at DESC
-		LIMIT $3 OFFSET $4`
+	   SELECT job_id, request, created_at, image_name
+	   FROM composes
+	   WHERE org_id = $1
+	   AND CURRENT_TIMESTAMP - created_at <= $2
+	   AND deleted = FALSE
+	   AND request->'image_requests'->0->>'image_type' <> ALL($3)
+	   ORDER BY created_at DESC 
+	   LIMIT $4 OFFSET $5;`
```

Comment:
Can you get the indentation right? It's 2 tabs, also no need to append `;` at the end.